tests/tls/test_tls.py

```diff
@@ -1,62 +1,39 @@
 import os
-from pathlib import Path
 
 import pytest
 from urllib3.exceptions import SSLError
 
 from clickhouse_connect import get_client
 from clickhouse_connect.driver.common import coerce_bool
 from clickhouse_connect.driver.exceptions import OperationalError
+from tests.helpers import PROJECT_ROOT_DIR
 
-cert_dir = f'{Path(__file__).parent}/'
+# See .docker/clickhouse/single_node_tls for the server configuration
+cert_dir = f'{PROJECT_ROOT_DIR}/.docker/clickhouse/single_node_tls/certificates/'
 host = 'server1.clickhouse.test'
 
-"""
-Sample openSSL section for clickhouse server configuration using the certificates in this directory
-
-<openSSL>
-    <server>
-        <certificateFile>clickhouse_test_server.crt</certificateFile>
-        <privateKeyFile>clickhouse_test_server.key</privateKeyFile>
-        <verificationMode>strict</verificationMode>
```

Comment:
Even with the certificates provided in this folder, I could not get it working in strict mode. So, in the `.docker/clickhouse/single_node_tls/config.xml`, it is `relaxed` now.